README.md

```diff
@@ -88,6 +88,7 @@ Any contribution is welcome!
 * [Animation Nodes](https://github.com/JacquesLucke/animation_nodes) - A node based visual scripting system designed for motion graphics in Blender
 * [ArmorPaint](https://armorpaint.org/) - A stand-alone software designed for physically-based texture painting
 * [Blender](https://blender.org) - Modeling and animation
+  * [Blender Awesome List](https://github.com/agmmnn/awesome-blender) - Mode addon and resources around blender in this awesome list.
```

Comment:
"Mode"? or did you mean "More"? 🤔

Also Titlecase "Blender" while you're there please.